Sming/Makefile-windows.mk

```diff
@@ -9,7 +9,7 @@ SDK_BASE	?= $(ESP_HOME)/ESP8266_SDK
 SDK_TOOLS	 ?= $(ESP_HOME)/utils
 
 # Other tools mappings
-ESPTOOL		 ?= $(SDK_TOOLS)/esptool.exe
+ESPTOOL		 ?= $(SDK_TOOLS)/ESP8266/esptool.exe
```

Comment:
Needed for correctly flashing. Since flashing is part of testing I considered this blocking issue and needed urgent fix...